opm/autodiff/MissingFeatures.cpp

```diff
@@ -22,12 +22,30 @@
 #include <opm/autodiff/MissingFeatures.hpp>
 #include <unordered_set>
 #include <string>
+#include <map>
 
 
 namespace Opm {
 
 namespace MissingFeatures {
 
+    template<typename T>
+    struct PartiallySupported {
+        int item;
```

Comment:
> My concern is that in the json format for these keywords, the options short cut(that is option name) are somehow different to Eclipse Manual

Yes - I see your point - I would still prefer that you use the string names:

1. Feel free to submit PR to parser changing the names to the Eclipse names.
2. Please use the compiled in named constants (we should be better at using these ...):
```C++
#include <opm/parser/eclipse/Parser/ParserKeywords/C.hpp>

string_options = { {ParserKeywords::COMPORD::keywordName , PartiallySupported<std::string>{1, ParseKeywords::COMPORD::TRACK::itemName}},

```

With some template trickery I guess it should be possible just use `ParserKeywords::COMPORD` and `ParserKeywords::COMPORD::TRACK`